code/datums/supply_packs.dm

```diff
@@ -1620,6 +1620,13 @@ ABSTRACT_TYPE(/datum/supply_packs/misc)
 		containertype = /obj/storage/crate/packing
 		containername = "Animal Import Kit"
 
+	mothroach
```

Comment:
Indentation here is messed up, and it should have a category, probably the same as the avian import kit or the honey production kit,

Reply:
Should be fixed in the Fixed mothroach cargo kit commit just now